srcpkgs/duplicity/template

```diff
@@ -12,19 +12,13 @@ license="GPL-2.0-or-later"
 homepage="http://duplicity.nongnu.org/"
 changelog="http://duplicity.nongnu.org/vers8/CHANGELOG"
 distfiles="http://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
-checksum=2cfcd212834979751310d9df1a954b949f86b084efd5ad1f97dd744780ef9d1b
+checksum=a273b1e74d8d0b7825572f51be9c8284e579961e01f4fc1473048e95dce49984
 
 post_extract() {
 	chmod 0644 bin/*.1
 }
 
-do_check() {
-	# Disable as tests don't work
-	# tests look for pytest,instead of pytest{2,3}
-	:
-}
-
 post_install() {
-	# drop doc
-	rm -rf ${DESTDIR}/usr/share/doc
+	# Remove docs
+	rm -rf usr/share/doc
```

Comment:
This doesn't do what you think; the working directory for `post_install` is `${wrksrc}` or, if defined, `${build_wrksrc}`; you are removing `usr/share/doc` from the build tree (which probabaly doesn't exist anyway), not the package destination.

Reply:
If at all possible, I'd like this to run with `rm -r` only, so it can catch the possibility of the file no longer existing. Most packages shouldn't be write protected, so I think it would work.

That said, why are docs being removed?

Reply:
Seems it was possibly removed before due to containing just these:
./usr/share/doc/duplicity-0.8.17/README.md
./usr/share/doc/duplicity-0.8.17/README-TESTING.md
./usr/share/doc/duplicity-0.8.17/README-REPO.md
./usr/share/doc/duplicity-0.8.17/README-LOG.md
./usr/share/doc/duplicity-0.8.17/COPYING
./usr/share/doc/duplicity-0.8.17/CHANGELOG.md
./usr/share/doc/duplicity-0.8.17/AUTHORS

So sorta, IMO, worthless as it doesn't contain things just as manpages and other helpful docs

Reply:
Fair enough.